src/explore-education-statistics-frontend/src/services/permalinkService.ts

```diff
@@ -47,22 +50,163 @@ export interface Permalink {
   created: string;
   fullTable: FullTable;
   configuration: {
-    tableHeadersConfig?: TableHeadersFormValues;
+    tableHeadersConfig: TableHeadersFormValues;
   };
 }
 
+const reverseMapTableHeadersConfig = (
```

Comment:
@bennettstuart @jamescutts I'd suggest potentially raising an "Improvement" Jira ticket for taking another look at this area in the future at some point, to possibly amend the metadata that we get back from the endpoints so that we don't need to do this metadata wrangling on the frontend to cater for Permalinks.

Woudn't want to hold up this PR from going in now though.

Reply:
Added a jira ticket in PostMVPBacklog
https://alpha-dfe-data.atlassian.net/browse/DFE-1386